sources/ClangSharp.PInvokeGenerator/PInvokeGenerator.VisitDecl.cs

```diff
@@ -501,12 +501,15 @@ private void VisitFunctionDecl(FunctionDecl functionDecl)
         var name = GetRemappedCursorName(functionDecl);
 
         var cxxMethodDecl = functionDecl as CXXMethodDecl;
+        uint overloadCount = 0;
 
         if (cxxMethodDecl is not null and CXXConstructorDecl)
         {
             var parent = cxxMethodDecl.Parent;
             Debug.Assert(parent is not null);
             name = GetRemappedCursorName(parent);
+
+            overloadCount = GetOverloadCount(cxxMethodDecl);
```

Comment:
Is this sufficient?

This will only set overload count for constructor decls, when presumably we want it for all cxxMethodDecl instead.

Reply:
you are absolutely right and I read that entirely wrong. fixed now.